plugins/modules/vmware_vc_infraprofile_info.py

```diff
@@ -0,0 +1,299 @@
+      type: str
+      required: False
+    api:
+      description:
+      - which API needs to be executed
+      type: str
+      required: False
+      choices: [ list, export, import, validate ]
+    config_path:
+      description:
+      - Config file path which contains infra profile config JSON data, supports both relative and absolute path.
+      - This parameter is required only when C(import),C(validate) APIs are being used.
+      type: str
+      required: False
+    profiles:
+      description:
+      - A list of profile names to be exported, imported, and validated.
+      - This parameter is not required while running for List API, not for C(export),C(import) and C(validate).
+      type: str
+      required: False
+    description:
+      description:
+      - Description of about encryption or decryption key.
+      type: str
+      required: False
+extends_documentation_fragment:
+- community.vmware.vmware_rest_client.documentation
+'''
+
+EXAMPLES = r'''
+- name: Get information about VC infraprofile
+  vmware_vc_infraprofile_info:
+    hostname: '{{ vcenter_hostname }}'
+    username: '{{ vcenter_username }}'
+    password: '{{ vcenter_password }}'
+    validate_certs: False
+  delegate_to: localhost
+
+- name: export Vcenter appliance infra profile config
+  vmware_vc_infraprofile_info:
+    hostname: '{{ vcenter_hostname }}'
+    username: '{{ vcenter_username }}'
+    password: '{{ vcenter_password }}'
+    validate_certs: False
+    api: "export"
+    profiles: "ApplianceManagement"
+  delegate_to: localhost
+
+- name: validate Vcenter appliance infra profile config
+  vmware_vc_infraprofile_info:
+    hostname: '{{ vcenter_hostname }}'
+    username: '{{ vcenter_username }}'
+    password: '{{ vcenter_password }}'
+    validate_certs: False
+    validate_certs: False
+    api: "validate"
+    profiles: "ApplianceManagement"
+    config_path: "export.json"
+
+- name: import Vcenter appliance infra profile config
+  vmware_vc_infraprofile_info:
+    hostname: '{{ vcenter_hostname }}'
+    username: '{{ vcenter_username }}'
+    password: '{{ vcenter_password }}'
+    validate_certs: False
+    validate_certs: False
+    api: "export"
+    profiles: "ApplianceManagement"
+    config_path: "export.json"
+  delegate_to: localhost
+  '''
+
+RETURN = r'''
+  ok: [localhost] => {
+    "list_infra": {
+        "changed": false,
+        "failed": false,
+        "infra_configs_list": [
+            {
+                "info": "ApplianceManagement",
+                "name": "ApplianceManagement"
+            },
+            {
+                "info": "ApplianceNetwork",
+                "name": "ApplianceNetwork"
+            },
+            {
+                "info": "Authentication & Authorization Management",
+                "name": "AuthManagement"
+            }
+        ]
+    }
+}
+
+ok: => {
+    "export_infra": {
+        "changed": false,
+        "export_config_json":"json exported to file"
+        }
+        }
```

Comment:
```suggestion
   export_infra: 
      description: A message about the exported file
      returned: On success with API set as "export"
      sample: {
        "export_config_json":"json exported to file"
      }
```

Same for the two below.